testplan/runnable/interactive/base.py

```diff
@@ -861,8 +859,9 @@ def _display_connection_info(self):
             )
 
         self.logger.user_info(
-            "\nInteractive Testplan web UI is running. Access it at:\n%s",
-            networking.format_access_urls(host, port, "/interactive/"),
+            "\nInteractive Testplan web UI is running. Access it at: %s:%s/interactive",
+            platform.node(),
```

Comment:
platform.node might be not fully qualified:https://docs.python.org/3/library/platform.html#platform.node

Reply:
Shall I use `getfqdn` instead? https://docs.python.org/3/library/socket.html#socket.getfqdn